README.md

```diff
@@ -198,9 +197,11 @@ DISABLE_DATABASE_ENVIRONMENT_CHECK=1
 ```
 
 Basically an especially inconvenient confirmation dialogue.  For safety's sake, the full command that skips that warning can be constructed by the user as needed, so as to prevent unintentional copy/paste dismissal when the user might be administering multiple deployment environments simultaneously.  Inadvertent database wipes are no fun.
-* `db:reset` will make sure db is migrated and seeded.  If you want to be extra sure: `RAILS_ENV=[environment] bundle exec rake db:migrate db:seed`
```

Comment:
this felt redundant, and unnecessary given that `rake db:reset` runs `rake db:seed` as part of what it does, and works just fine.